pandas/tests/frame/test_alter_axes.py

```diff
@@ -871,6 +871,29 @@ def test_rename_bug2(self):
                              columns=["a"])
         tm.assert_frame_equal(df, expected)
 
+    def test_rename_errors(self):
+        # GH 13473
+        # rename now works with errors parameter
+
+        # Error has to be thrown and is thrown
+        df = DataFrame(columns=['A', 'B', 'C', 'D'])
```

Comment:
We typically place these types of tests in a separate function with `_tests` appended to the end. Comment also not necessary

Reply:
Do you mean only the first added test (e.g. the test which checks whether an error is thrown) or all tests together? 

Moreover, I am unable to find a test which uses such naming pattern; would you mind giving me an example of one of these tests?

Reply:
Sorry I meant to say `_raises`. Here's one:

https://github.com/pandas-dev/pandas/blob/d10bbce349b293e9f753d799cd2ad032d5ec7333/pandas/tests/frame/test_alter_axes.py#L379

Reply:
I see, that definitely clears things up! Thanks!